src/components/avatar/Avatar.js

```diff
@@ -4,9 +4,9 @@ import Image from 'react-bootstrap/Image';
 
 import defaultAvatar from '../../assets/images/avatar-default.webp';
 
-const Avatar = ({ imgSrc = defaultAvatar, avatarSize = 'small' }) => (
+const Avatar = ({ imgSrc, avatarSize = 'small' }) => (
   <div className={avatarSize === 'small' ? 'smallavatarContainer' : 'largeavatarContainer'}>
-    <Image alt="userImage" src={imgSrc} roundedCircle />
+    <Image alt="userImage" src={imgSrc !== '' ? imgSrc : defaultAvatar} roundedCircle />
```

Comment:
We do not need the imgSrc !== '' condition or the ternary operator. You can directly say imgSrc || defaultAvatar. This will also take care of the case where imgSrc is null or undefined.

Reply:
Good catch